src/main/java/com/loadero/http/TestTypeAdapter.java

```diff
@@ -93,8 +93,8 @@ public void write(JsonWriter jsonWriter, Test test) throws IOException {
         jsonWriter.beginObject();
         jsonWriter.name(NAME).value(test.getName());
         jsonWriter.name(SCRIPT).value(test.getScript());
-        jsonWriter.name(START_INTERVAL).value(test.getStartInterval().toString());
-        jsonWriter.name(PARTICIPANT_TIMEOUT).value(test.getParticipantTimeout().toString());
+        jsonWriter.name(START_INTERVAL).value(test.getStartInterval().getSeconds());
```

Comment:
may need to add reference to this MR in the GH issue

Reply:
Added a comment there